server/modules/user/manager/userAnalysisManager.js

```diff
@@ -1 +1,24 @@
-export { insertOrUpdateUserAnalysis } from '../repository/userAnalysisRepository'
+import { db } from '@server/db/db'
```

Comment:
should this manager to to analysis module?

Reply:
ok